doc/decisions/0007-feature-toggles-development.md

```diff
@@ -0,0 +1,92 @@
+# 0007. Feature toggles development
+
+| Date | Tags |
+|---|---|
+| 2018-05-25 | development |
+
+
+## Status
+
+Proposed
+
+## Context
+
+This change describes a methodology of development which we believe will help
+us increase the confidence and speed in the development process.
+
+### What problem are we trying to solve
+
+Currently the time between a PR is sent and the code gets merged is between 1 day
+to in some case 4 to 5 days.
```

Comment:
Do we have a target in mind?

Reply:
Like: 70% of the PRs are merged within the same day?

Reply:
I am not sure, hard to say, as only new-features really can benefits from this, I think still the majority will go through manual QA.

Reply:
One thing is that then it implies that a developer should mark the PR that needs to be manually checked with a label for QA to know.
I'd talk about that with the QA team too.
In the more extreme scenario, we can even skip PR testing for more thorough nightly testing (except some PRs).

Reply:
yes, a label needs to be added. I already mentioned this approach on slack with the test team, they have some concerns but they agree overall I believe. But I will ask them to review this once the conversation among developers settles, so that we are happy with the changes.